src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/Backup.kt

```diff
@@ -45,7 +67,25 @@ open class Backup : ZipTask() {
         from(aem.config.localInstanceOptions.root)
     }
 
+    private fun fileTransfer(url: String): FileTransfer {
```

Comment:
I am sure that we want to go with reducing and removing `aem.config` and moving properties to specialized services available directly from `aem` extension.

e.g on `develop` branch there is already `aem.environment` which is a service for controlling Docker env; we have `aem.notifier` notifier facade for GUI notifications, etc

next step will be to introduce `aem.fileTransfer` which will have properties https://github.com/Cognifide/gradle-aem-plugin/blob/master/src/main/kotlin/com/cognifide/gradle/aem/common/file/resolver/ResolverOptions.kt#L8-L24 but I am thinking that we could integrate our code after merging you changes first.

Reply:
OK